packages/GA4Client/src/common.js

```diff
@@ -64,6 +64,31 @@ Common.prototype.truncateEventAttributes = function (eventAttributes) {
     );
 };
 
+Common.prototype.standardizeParameters = function (parameters) {
+    var standardizedParameters = {};
+    Object.keys(parameters).forEach(key => {
+        var standardizedKey = (' ' + key).slice(1);
+
+        standardizedKey = standardizedKey.replace("[^a-zA-Z0-9_]".toRegex(), "_")
+        for (forbiddenPrefix in forbiddenPrefixes) {
```

Comment:
I think `forbiddenPrefixes` is defined in the `commerce-handler` but it isn't being imported. I would just add them to this file.

Reply:
Also, you should be doing `for of` instead of `for in` since `forbiddenPrefixes` is an array.